pkg/antctl/raw/multicluster/common/common.go

```diff
@@ -0,0 +1,415 @@
+	clusterClaim := newClusterClaim(clusterID, namespace, false)
+	unstructuredClusterClaim, _ = runtime.DefaultUnstructuredConverter.ToUnstructured(clusterClaim)
+
+	if createErr = k8sClient.Create(context.TODO(), clusterClaim); createErr != nil {
+		if !apierrors.IsAlreadyExists(createErr) {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create ClusterClaim \"%s\": %v\n", multiclusterv1alpha2.WellKnownClusterClaimID, createErr)
+			return createErr
+
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" already exists in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimID, namespace)
+		createErr = nil
+	} else {
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" created in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimID, namespace)
+		*createdRes = append(*createdRes, unstructuredClusterClaim)
+	}
+
+	clusterClaim = newClusterClaim(clusterset, namespace, true)
+	unstructuredClusterClaim, _ = runtime.DefaultUnstructuredConverter.ToUnstructured(clusterClaim)
+
+	if createErr = k8sClient.Create(context.TODO(), clusterClaim); createErr != nil {
+		if !apierrors.IsAlreadyExists(createErr) {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create ClusterClaim \"%s\": %v\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, createErr)
+			return createErr
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" already exists in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, namespace)
+		createErr = nil
+	} else {
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" created in Namespace %s\n", multiclusterv1alpha2.WellKnownClusterClaimClusterSet, namespace)
+		*createdRes = append(*createdRes, unstructuredClusterClaim)
+	}
+
+	return nil
+}
+
+func CreateClusterSet(cmd *cobra.Command, k8sClient client.Client, namespace string, clusterset string,
+	leaderServer string, secret string, memberClusterID string, leaderClusterID string, leaderClusterNamespace string, createdRes *[]map[string]interface{}) error {
+	var unstructuredClusterSet map[string]interface{}
+	clusterSet := newClusterSet(clusterset, namespace, leaderServer, secret, memberClusterID, leaderClusterID, leaderClusterNamespace)
+	unstructuredClusterSet, _ = runtime.DefaultUnstructuredConverter.ToUnstructured(clusterSet)
+
+	if err := k8sClient.Create(context.TODO(), clusterSet); err != nil {
+		if !apierrors.IsAlreadyExists(err) {
+			fmt.Fprintf(cmd.OutOrStdout(), "Failed to create ClusterSet \"%s\": %v\n", clusterSet.Name, err)
+			return err
+		}
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterSet \"%s\" already exists in Namespace %s\n", clusterSet.Name, clusterSet.Namespace)
+	} else {
+		fmt.Fprintf(cmd.OutOrStdout(), "ClusterSet \"%s\" created in Namespace %s\n", clusterSet.Name, clusterSet.Namespace)
+		*createdRes = append(*createdRes, unstructuredClusterSet)
+	}
+
+	return nil
+}
+
+func deleteClusterClaim(cmd *cobra.Command, k8sClient client.Client, namespace string) error {
```

Comment:
I think we should add "s" to all delete func names, as they can delete more than one resources.

Reply:
Added "s" to all functions.